### PR TITLE
Cyblethreatintel

### PR DESCRIPTION
**Pull request by cyble-dev**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
...N2ExLjYyLDEuNjIsMCwwLDEsLjM0LjQ3TDI4LjQsNjAuNmMuMTcuMjguMzYuNTUuNTUuODJsLS4xMi4xWiIvPjxwYXRoIGQ9Ik0zMi44Niw0NS43NWMtLjE3LTMuMjQtLjM0LTYuODktLjUyLTEwLjEuMzMuMTkuNTkuMzcuODYuNTFsNi43MiwzLjQ4YTcsNywwLDAsMSwuNjMuMzdzLS41NC4zMS0uODQuNTFjLS44OC41Ny0xLjc4LDEuMTMtMi42NSwxLjczbC0uNzYuNTNjLS42LjQ1LTEuMTkuOTItMS43NSwxLjQyUzMzLjQzLDQ1LjI0LDMyLjg2LDQ1Ljc1WiIvPjxwYXRoIGQ9Ik0xOS4yNyw0MiwxOCw0Ni42NSw5LjMxLDQwLjc0bDAtLjE5WiIvPjxwYXRoIGQ9Ik0zMywyMS43NWMtLjMtLjA4LTguODItMi44My05LjA3LTNzLS4xMi0uMywwLS40NWExLjM1LDEuMzUsMCwwLDEsLjQ3LS4zNSw1LjI5LDUuMjksMCwwLDEsMS4zMi0uMzFjLjMxLDAsLjYyLS4wOC45NC0uMTEsMS4wNi0uMTEsMi4xMy0uMTIsMy4yLS4xNGExLjQ0LDEuNDQsMCwwLDEsMS4zNywxQzMxLjczLDE5LjMzLDMyLjkxLDIxLjQ4LDMzLDIxLjc1WiIvPjxwYXRoIGQ9Ik0zNy4yNCwxOUMzNC45MSwyMS4xNywzNCwyMS45LDM0LDIxLjljLS42NS0xLjIxLTEuMjgtMi4zNC0xLjg5LTMuNDktLjE3LS4zMy4xNS0uNzMuNjYtLjc4YTMsMywwLDAsMSwuNzguMDdsMi40OS40QzM2LjU0LDE4LjIsMzcuODUsMTguNDUsMzcuMjQsMTlaIi8+PHBhdGggZD0iTTM5LjcsMjQuNDRhLjE1LjE1LDAsMCwxLDAsLjA5YzAsLjQyLS43Ni41Ny0xLjA2LjU5LS43Mi4wNi0xLjQ0LjA5LTIuMTYuMXMtMi4wOS4yOC0yLjc0LS40M2ExLjI5LDEuMjksMCwwLDEtLjIyLTEsNy40MSw3LjQxLDAsMCwxLC4xMS0xYzEuMS0uNTIsMi4xNC4xMiwzLjE5LjUxbDEuNzUuNjZDMzguNzgsMjQsMzkuNTgsMjQuMTcsMzkuNywyNC40NFoiLz48L3N2Zz4=
```
Comment (bziser) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml:88]: What is this image?

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/Cyble_Logo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...,20.93,41.45Z"/><path d="M56,34.9l-7.57-7.61c-.2-.21-.55-.41-.29-.74a1.13,1.13,0,0,1,.68-.31,7.31,7.31,0,0,1,3.45.07,11.61,11.61,0,0,1,7.33,7.23,11.75,11.75,0,0,1,0,7.23c-.16.48-.64,1.67-.64,1.67s-.51-.38-.6-.47c-.71-.66-2.39-2.35-2.39-2.35Z"/><path d="M16.86,54.07,18.13,48c1.31.89,2.55,1.72,3.78,2.57a1.62,1.62,0,0,1,.34.47L28.4,60.6c.17.28.36.55.55.82l-.12.1Z"/><path d="M32.86,45.75c-.17-3.24-.34-6.89-.52-10.1.33.19.59.37.86.51l6.72,3.48a7,7,0,0,1,.63.37s-.54.31-.84.51c-.88.57-1.78,1.13-2.65,1.73l-.76.53c-.6.45-1.19.92-1.75,1.42S33.43,45.24,32.86,45.75Z"/><path d="M19.27,42,18,46.65,9.31,40.74l0-.19Z"/><path d="M33,21.75c-.3-.08-8.82-2.83-9.07-3s-.12-.3,0-.45a1.35,1.35,0,0,1,.47-.35,5.29,5.29,0,0,1,1.32-.31c.31,0,.62-.08.94-.11,1.06-.11,2.13-.12,3.2-.14a1.44,1.44,0,0,1,1.37,1C31.73,19.33,32.91,21.48,33,21.75Z"/><path d="M37.24,19C34.91,21.17,34,21.9,34,21.9c-.65-1.21-1.28-2.34-1.89-3.49-.17-.33.15-.73.66-.78a3,3,0,0,1,.78.07l2.49.4C36.54,18.2,37.85,18.45,37.24,19Z"/><path d="M39.7,24.44a.15.15,0,0,1,0,.09c0,.42-.76.57-1.06.59-.72.06-1.44.09-2.16.1s-2.09.28-2.74-.43a1.29,1.29,0,0,1-.22-1,7.41,7.41,0,0,1,.11-1c1.1-.52,2.14.12,3.19.51l1.75.66C38.78,24,39.58,24.17,39.7,24.44Z"/></svg>
```
Comment (bziser) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/Cyble_Logo.svg:1]: Logo should match the requirements (120x50 px PNG, transparent background, <10kB in Size)

File: Packs/CybleThreatIntel/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "CybleThreatIntel",
+    "description": "Cyble Threat Intelligence for Vision Users. Must have access to Vision Taxii feed to access the threat intelligence.",
+    "support": "xsoar",
```
Comment (bziser) [inline Packs/CybleThreatIntel/pack_metadata.json:4]:
```suggestion
    "support": "partner",
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                # request was succesful
+                return_results(resp)
+
+        elif demisto.command() == 'cyble-vision-fetch-taxii':
+            # fetch events using taxii service
+
+            if 'start_date' not in args.keys():
+                args['start_date'] = datetime.today().strftime('%Y-%m-%d')
+            if 'end_date' not in args.keys():
+                args['end_date'] = datetime.today().strftime('%Y-%m-%d')
```
Comment (bziser) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:137]: Why not to do these lines before the commands and so prevent duplicates?

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+            if args['token'] is not None:
+                command_results = CommandResults(
+                    outputs_prefix='CybleIntel.Threat',
+                    outputs_key_field='details',
+                    outputs=cyble_fetch_taxii(client, "POST", args)
+                )
```
Comment (bziser) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:144]: You can do this in the `cyble_fetch_taxii` function, and so the `cyble_fetch_taxii` function will return `CommandResults`

**Review by bziser — COMMENTED** [review]
Thank you for your conribution, here are my comments from an initial review.

**Comment by cyble-dev** [comment]
Have updated the review suggestions

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml (hunk 1 above)
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml]: it is the logo, YML download from XSOAR had this

**Review by bziser — APPROVED** [review]

**Pull request merged**